go/vt/wrangler/traffic_switcher.go

```diff
@@ -58,9 +59,14 @@ const (
 // accessType specifies the type of access for a shard (allow/disallow writes).
 type accessType int
 
+// tableLockAction specifies if you want to lock or unlock tables
+type tableLockAction int
```

Comment:
Let's make this a strongly typed `iota` to have better clarity, then have something like `tableLockActionAquire` and `tableLockActionRelease`. That way we don't need to define `tableLockAcquire = tableLockAction(0)` etc as integers.